### ISSUE TITLE
Menu - vue-router integration is buggy

### ISSUE THREAD

**Issue by mrin9** [AT-UI/at-ui#99]
### Steps to reproduce

Set up vue-router with few navigations
Create few menu-items with router integration (provide router paths in ```to``` prop)
Select an item from menu which will make it active (blue vertical bar on side) 
Now type a different path on the browser's url, this will make the other associated item active in the menu, but will not remove the previously selected item (now I have a menu showing two items selected) 

### Which browser?
All

### What is expected?
upon typing a valid path, on browser's url, The menu must remove the previous selected item

### What is actually happening?
The previous selected item is not removed

### Reproduction Link
Sorry cant create a reproduction link as the use case is tied to browser's url.

**Comment by dsevillamartin** [comment]
I got this bug as well, and for me it seems that AT UI is not only checking for exact route paths, but also similar (like starting with, etc...).

Here's the classes on my Home menu item pointing to `/`:
![image](https://user-images.githubusercontent.com/6401250/40880901-aadb2572-6687-11e8-80f3-b3b6e4f9fd83.png)

Here are the classes for the same element when I click on another item and the path changes:
![image](https://user-images.githubusercontent.com/6401250/40880904-b84b615e-6687-11e8-9e9f-a8e689aafecd.png)

The `router-link-exact-active` class is dropped, but `router-link-active` remains, which led be to the conclusion illustrated at the beginning of this comment.

This is a nice feature, however there should be an option to disable it when needed, which usually includes the home (`/`) path.

**UPDATE**: `router-link-exact-active` is provided by `vue-router` (see https://router.vuejs.org/api/#exact-active-class), so the logic AT UI uses is definitely faulty here, whatever it's supposed to be doing.

**Comment by SeedyROM** [comment]
@datitisev 
Currently dealing with this bug, I tried solving at the AtMenuItem component level. No luck, I'll keep everyone posted and create a PR when done.

The styling also seems to be causing the issue, not the logic itself from what I can see. I went through and completely modified the AtMenuItem to have an exact prop, the styling for active turns on regardless of if the item actually is.

**Update:** I thought I found the issue, no luck.

**Comment by SeedyROM** [comment]
In case anyone finds this thread, the issue is actually with assuming you can set the `to` prop on `AtMenuItem` with only a string path, and that it will match exactly _(It doesn't)_.

Setting the `to` prop like so `:to="{ name: 'home' }` matches correctly instead of doing `to="/"`.
The `AtMenuItem` actually actually maps over the keys in the `to` prop and injects them into the `router-link`. Not sure why this solves the problem exactly, but it's a quick fix.

**TLDR;**

Set the `to` prop with the routes name as an object (`:to="{ name: 'home' }`) instead of using (`to="/"`)
Obviously replace home with whatever route name you want.

**Comment by SeedyROM** [comment]
See PR https://github.com/AT-UI/at-ui/pull/143/, this might solve your issue. Still in development.
